docs/cross-references.md

```diff
@@ -171,6 +171,12 @@ See [](#my-math-label) for an equation!
 
 `````{tip}
 # Labelling dollar math
+
+
+````{warning}
+The following syntax for labelling dollar-math equations is in `beta`, as it is not yet described by the [MyST Specification](https://myst-spec.readthedocs.io/en/latest/index.html). As such, it may change in the future.
+````
```

Comment:
I think that we might actually just want remove dollar math labels from the documentation, there are other ways to label that are more consistent (i.e. a directive or a target assignment `()=`)